pulsar-zookeeper-utils/src/main/java/org/apache/pulsar/stats/CacheMetricsCollector.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pulsar.stats;
+
+import lombok.experimental.UtilityClass;
+
+@UtilityClass
+public class CacheMetricsCollector {
+
+    public static final io.prometheus.client.cache.caffeine.CacheMetricsCollector CAFFEINE = new io.prometheus.client.cache.caffeine.CacheMetricsCollector()
+            .register();
+
+    public static final io.prometheus.client.guava.cache.CacheMetricsCollector GUAVA = new io.prometheus.client.guava.cache.CacheMetricsCollector()
```

Comment:
Where is this being used?

Reply:
I was initially planning to instruments also Guava caches, though the remaining ones we have are not very interesting, so dropped the ball. 

I have removed guava now. PTAL.